yos/list_test.go

```diff
@@ -85,6 +89,8 @@ func TestListDir(t *testing.T) {
 		if _, err := ListFile(path); err == nil {
 			t.Errorf("ListFile(%q) got no error, diff from expected", path)
 			return
+		} else {
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [if block ends with a return statement, so drop this else and outdent its block (move short variable declaration to its own line if necessary)](https://app.codacy.com/manual/an9an63/gut/pullRequest?prid=4927519)